example-dotnet/Program.cs

```diff
@@ -11,8 +22,107 @@ static void Main(string[] args)
             Say.hello("example");
             Console.WriteLine("Hello World!");
 
+	    // Usage of BCrypt.Net API
+
+            // The following API call is meant to demonstrate
+            // vulnerable methods (the simplest case).
+              
             var hashedPassword = BCrypt.Net.BCrypt.HashPassword("my-secret-password");
             BCrypt.Net.BCrypt.Verify("wrong-password", hashedPassword);
+
+            // The next API calls to four libraries are meant to
+            // demonstrate update advisor and auto pull request
+            // features.
+            //
+            // The first two are the cases where the update versions
+            // are of the same frameworks as the currently-used
+            // version.
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | Dependency         | Current      | Current        | Update    | Update          | Breaking |
+            // |                    | Version      | Framework(s)   | Version   | Framework(s)    | Update   |
+            // +====================+==============+================+===========+=================+==========+
+            // | recurly-client-api | 1.8.0        | net35          | 1.8.1 or  | net35           | No       |
+            // |                    |              |                | 1.11.4    |                 |          |
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | DotNetZip          | 1.10.0       | net20          | 1.11.0    | net20           | Maybe    |
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            //
+            // The other two are the cases where the update versions
+            // have different frameworks than the current versions.
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | Dependency         | Current      | Current        | Update    | Update          | Breaking |
+            // |                    | Version      | Framework(s)   | Version   | Framework(s)    | Update   |
+            // +====================+==============+================+===========+=================+==========+
+            // | SharpZipLib        | 1.0.0-alpha1 | netstandard1.3 | 1.0.0-rc1 | net45,          | Maybe    |

```

Comment:
What I meant was that we should change the "Maybe"s to "Yes"es. It's closer to what we actually mean ("Yes, with assumptions" rather than "Maybe, not sure"). This is also as certain as we can be using current methods, so there's no point equivocating.

```suggestion
            // | SharpZipLib        | 1.0.0-alpha1 | netstandard1.3 | 1.0.0-rc1 | net45,          | Yes      |
```

Reply:
@dariusf Done. Thanks!